spec/abstractops.html

```diff
@@ -29,6 +29,119 @@ <h1>IterableToListOfType ( _items_, _elementTypes_ )</h1>
     </emu-alg>
   </emu-clause>
 
+  <emu-clause id="sec-isodatetoepochdays" type="abstract operation">
+    <h1>
+      ISODateToEpochDays (
+        _year_: an integer,
+        _month_: an integer,
+        _date_: an integer,
+      ): an integer
+    </h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>It calculates a number of days.</dd>
+    </dl>
+    <emu-alg>
+      1. Let _ym_ be _year_ + (floor(_month_) / 12).
+      1. Let _mn_ be _month_ modulo 12.
+      1. Find a time value _t_ such that EpochTimeToEpochYear(_t_) is _ym_, EpochTimeToMonthInYear(_t_) is _mn_, and EpochTimeToDate(_t_) is 1.
```

Comment:
I think using the "time value" type here isn't great, because it still invites the problem that I identified in https://github.com/tc39/ecma262/issues/1087#issuecomment-1120110847 — it could be out of range.
However, we do need a time value to pass to Day in the next step, and even if we did 𝔽(_t_) that could still be ±∞. So I'm afraid we might need to introduce a mathematical version of Day as well.